swift/apple/FirezoneKit/Sources/FirezoneKit/Helpers/Log.swift

```diff
@@ -97,20 +97,32 @@ private final class LogWriter {
       logger.error("Log directory isn't acceptable!")
       return nil
     }
-    self.logFileURL = folderURL
+
+    let logFileURL = folderURL
       .appendingPathComponent(dateFormatter.string(from: Date()))
       .appendingPathExtension("log")
 
     // Create log file
-    guard fileManager.createFile(atPath: self.logFileURL.path, contents: "".data(using: .utf8))
+    guard fileManager.createFile(atPath: logFileURL.path, contents: "".data(using: .utf8)),
+          let handle = try? FileHandle(forWritingTo: logFileURL),
+          let _ = try? handle.seekToEnd()
```

Comment:
Surprised Swift makes you do a seek instead of having a thing to open files in append mode, C-style.

Reply:
Yeah, tell me about it